### ISSUE TITLE
CVE-2023-35116: jackson-databind package versions before 2.16.0

### ISSUE THREAD

**Issue by hbornstein747** [IBM/ibm-cos-sdk-java#64]
See https://github.com/FasterXML/jackson-databind/issues/3972 and https://nvd.nist.gov/vuln/detail/CVE-2023-35116

Even after updating to suggested version for this CVE, it is still being reported as vumnerable and now requires yet another upgrade to jackson-databind 2.16.0.  This is related to https://github.com/IBM/ibm-cos-sdk-java/issues/63

Can it be done for the COS SDK?
cc: @tcherel

**Comment by avinash1IBM** [comment]
@hbornstein747 the jackson-databind team didn't agree that it is a vulnerability https://github.com/FasterXML/jackson-databind/issues/3972#issuecomment-1813589383, and even if we upgrade the package to 2.16.0, then the same vulnerability will be flagged again. Even the nvd site that you shared above added a note saying this 
`NOTE: the vendor's perspective is that this is not a valid vulnerability report, because the steps of constructing a cyclic data structure and trying to serialize it cannot be achieved by an external attacker.` 

So can you please close this ticket?

**Comment by hbornstein747** [comment]
@tcherel ^^^

**Comment by hbornstein747** [comment]
@avinash1IBM I agree that this is just churn and the vulnerability will likely come up again, but we (CP4D team at IBM) are being asked to update databind regardless.  I don't want to close the ticket until our securoty focal (@tcherel) can comment.

**Comment by avinash1IBM** [comment]
@hbornstein747 A new version of ibm-cos-sdk-java is released to fix this. Please close this issue
